modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js

```diff
@@ -454,13 +454,21 @@ AUI.add(
 
 					const contentBox = instance._contentBox;
 
-					const visibleRows = contentBox.all('.lfr-form-row:visible');
+					const allRowsNodes = contentBox.all('.lfr-form-row')._nodes;
 
-					const visibleRowsSize = visibleRows.size();
+					const visibleRows = [];
 
-					let deleteRow = visibleRowsSize > 1;
+					for (let i = 0; i < allRowsNodes.length; i++) {
+						if (allRowsNodes[i].checkVisibility()) {
+							visibleRows.push(allRowsNodes[i]);
+						}
+					}
+
+					const visibleRowsLength = visibleRows.length;
+
+					let deleteRow = visibleRowsLength > 1;
```

Comment:
I may be wrong here, but can't you just use .filter? We tend to try and avoid accessing "private" properties like `._node` since it has a leading underscore.

```js
const visibleRows = contentBox.all('.lfr-form-row').filter(
    node => node.getDOMNode().checkVisibility()
)
```

Not sure if this would make more sense.


I'm also fine merging without this change